docs/02-StewardForValidators.md

```diff
@@ -0,0 +1,31 @@
+# Validators Instructions for Setting Up Steward
+
+Steward is a side-car process that connects the Sommelier chain to the outside world. It provides an API for Strategy Providers to rebalance Cellars through the Sommelier validator set based on off-chain calculations. Additionally, it subsumes the functionality of gorc for managing the [Gravity bridge Orchestrator](https://github.com/PeggyJV/gravity-bridge/tree/main/orchestrator), and provides developer tools and features for developing and testing Cellars.
```

Comment:
This section is a copy-paste of the top-level README intro. Do we want to duplicate this? If we do, we should also make sure that they are both updated in tandem.

Reply:
I figured Validators would come to this doc more frequently than the README so it might be good to have it in both places. I don't see it changing much but it can go either way for me.